src/main/java/serwer/dane/KontrolerStanuGry.java

```diff
@@ -0,0 +1,46 @@
+package serwer.dane;
+
+import java.util.EnumMap;
+
+public class KontrolerStanuGry {
+    private StanGry stan;
+
+    public enum StanGry{
+        RUCH_BIALYCH,
+        RUCH_CZARNYCH,
+        SKONCZONA,
+        NIEROZPOCZETA,
+        PRZERWANA
+    }
```

Comment:
Tutaj można zastanowić się nad implementacją wzorca State, ale to już zależnie od tego ile czasu nam zostanie.

Reply:
taki był plan, po prostu trochę to zostawiłem